js/src/modals/ExecuteContract/executeContract.js

```diff
@@ -334,17 +399,25 @@ class ExecuteContract extends Component {
           });
       })
       .then((txhash) => {
-        this.setState({ sending: false, step: finalstep, txhash, busyState: 'Your transaction has been posted to the network' });
+        this.setState({
+          sending: false,
+          step: finalstep,
+          txhash,
+          busyState:
```

Comment:
I'd prefer to keep plain strings in the state and use `FormattedMessage` when rendering.